pysus/classes/sinan/__init__.py

```diff
@@ -0,0 +1,266 @@
+import os
+import re
+from ftplib import FTP
+from pathlib import Path
+from typing import List, Union
+
+import pandas as pd
+import pyarrow as pa
+import pyarrow.parquet as pq
+from dbfread import DBF
+from pysus.utilities.readdbc import dbc2dbf
+from sqlalchemy import DATE, INTEGER, NUMERIC, VARCHAR
+
+from .diseases import DISEASE_CODE
+from .typecast import COLUMN_TYPE
+
+
+class SINAN:
```

Comment:
@luabida if this refactor is going to break the PySUS current API, it must be discussed in more details first.